components/api-server/src/bin/api_server.rs

```diff
@@ -0,0 +1,100 @@
+use axum::{
+    Json,
+    Router,
+    extract::{Path, State},
+    http::StatusCode,
+    response::{
+        IntoResponse,
+        Sse,
+        sse::{Event, KeepAlive},
+    },
+    routing::{get, post},
+};
+use clap::Parser;
+use clp_rust_utils::package_config;
+use futures::{Stream, StreamExt};
+use thiserror::Error;
+use tracing_subscriber::{self, prelude::*};
+
+#[derive(Parser)]
+#[command(version, about)]
+struct Args {
+    #[arg(long)]
+    package_root: String,
```

Comment:
shall we call it `clp_home` to follow the convention in the CLP Package Python scripts?

---

in fact, we read such info from the env var `CLP_HOME` instead of from the command-line: https://github.com/y-scope/clp/blob/7c95e23fa5caf05ae3f3db56d90386f6fe868d3e/components/clp-package-utils/clp_package_utils/scripts/compress.py#L125

we do take `--config` from the arguments: https://github.com/y-scope/clp/blob/7c95e23fa5caf05ae3f3db56d90386f6fe868d3e/components/clp-package-utils/clp_package_utils/scripts/compress.py#L131-L137

were there reasons to derivate from the conventions used in the CLP Package Python scripts?